apps/admin-server/src/pages/projects/[project]/statuses/create.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { zodResolver } from '@hookform/resolvers/zod';
+import * as z from 'zod';
+import { useForm } from 'react-hook-form';
+
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { PageLayout } from '@/components/ui/page-layout';
+import { Heading } from '@/components/ui/typography';
+import { Separator } from '@/components/ui/separator';
+import { useRouter } from 'next/router';
+import useStatus from '@/hooks/use-statuses';
+import toast from 'react-hot-toast';
+
+const formSchema = z.object({
+  name: z.string(),
+  seqnr: z.coerce.number()
```

Comment:
Is seqnr verplicht voor het aanmaken van een status en wil je hier niet ook alle andere velden kunnen aanmaken?

Reply:
Seqnr is een verplicht veld inderdaad.

En wat betreft het tweede punt: er zijn meer objecten in de admin die je alleen in een simpele versie kunt aanmaken en pas in de update kunt uitbreiden.  Ook hier: prima om dat in het algemeen anders te doen maar dat valt buiten de scope van deze PR.